projects/ngrx.io/src/app/layout/footer/footer.component.html

```diff
@@ -17,7 +17,7 @@ <h3>{{node.title}}</h3>
 </div>
 
 <p>
-    Powered by the Community ©2015-2023.
+    Powered by the Community ©2015-2024.
```

Comment:
```suggestion
    Powered by the Community ©2015-2025.
```

Reply:
" Powered by the Community ©2015 - end of time." would look cool tho.